### ISSUE TITLE
Unable to calculate days difference

### ISSUE THREAD

**Issue by vresetnikov** [spencermountain/spacetime#353]
Hello,

It appears there is a missing type definition or perhaps a simple mismatch with what is written in the documentation:

The official spacetime documentation here on Github demonstrates that it is possible to calculate the difference between 2 dates in such a manner:

`s.diff(s.endOf('year'), 'days')`

However, attempting to follow the provided instructions makes Typescript raise the following issue:

![image](https://user-images.githubusercontent.com/54367758/205327665-0d514e55-8268-4f47-96f4-c353b591da28.png)

The code runs fine nonetheless, leading me to believe that perhaps this function parameter type was omitted by accident:

```ts
export type TimeUnit =
  | 'millisecond'
  | 'second'
  | 'minute'
  | 'quarterHour'
  | 'hour'
  | 'day'
  | 'week'
  | 'month'
  | 'quarter'
  | 'season'
  | 'year'
  | 'decade'
  | 'century'
  | 'date'
```

Changing the keyword from 'days' to 'day' eliminates the Typescript error and works fine as well. This problem has been encountered in ver. 7.2.0

**I would also like to take time to thank you for this library, everything else that performs date operations pales in comparison to this library and its intuitiveness and ease of use.** 

Thank you

**Comment by spencermountain** [comment]
thanks!
I'll add it to the next release
cheers

**Comment by spencermountain** [comment]
fixed in 7.3.0
thanks
